components/automate-ui/src/app/page-components/navbar/navbar.component.html

```diff
@@ -7,7 +7,9 @@
       <div class="navigation-menu">
         <div role="menuitem" class="nav-link"><a routerLink="/dashboards/event-feed" routerLinkActive="active" tabindex="0">Dashboards</a></div>
 
-        <div role="menuitem" class="nav-link"><a routerLink="/applications/service-groups" routerLinkActive="active" tabindex="0">Applications</a></div>
+        <app-authorized [anyOf]="['/auth/introspect', 'get']">
```

Comment:
@bcmdarroch @tyler-ball @msorens - I think I have this setup correctly, but Its doesn't seem to be working as I expect it to.  

If i'm reading the readme correctly, a `GET` call to the `/auth/introspect` endpoint should check all of the non parameterized endpoints
>1. GET - Does an "IntrospectAll", returning data for all non-parameterized endpoints
   (i.e. concrete endpoints like "/auth/teams" but not abstract endpoints like "/auth/users/{email}").

So I'm expecting this would check all of the users authorizations/permisssions, but I'm still seeing the Applications link shown when a user does not have access to Applications.  Does this look off to you?

Reply:
oh also tyler ball is a different engineer at chef, hes on the workstation team.. tyler cloke is the tyler on the auth team

Reply:
haha whoops!  def meant tyler cloke!  Thanks susan!